RevenueCat.podspec

```diff
@@ -24,6 +24,7 @@ Pod::Spec.new do |s|
   s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
 
   s.source_files = 'Sources/**/*.swift'
+  s.exclude_files = 'Sources/LocalReceiptParsing/ReceiptParser-only-files/*'
```

Comment:
We need some files specific to just the `ReceiptParser` module (right now the separate `LoggerType`).

Reply:
Should this be `**` to match within subdirectories? Just in case we ever addd directories to that folder

Reply:
Let me try

Reply:
This works 👍🏻 changed.